bundles/org.openhab.persistence.influxdb/README.md

```diff
@@ -34,6 +34,12 @@ This is very easy and you will find good documentation on it on the
 
 ## Configuration
 
+### Configuration via Web-Interface:
```

Comment:
```suggestion
### Configuration via web interface
```

Or perhaps use term "UI" like you did in header "### File based configuration (not needed when done via UI)" to not mix terms "UI" and "web interface"?